dev/session_metrics/lib/slurm_metrics/metrics_session_service.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module SlurmMetrics
+  # Service to manages retrieving and storing the user job metrics.
+  class MetricsSessionService
```

Comment:
If this handles the `SessionMetrics` class maybe naming this service class `SessionMetricsService` could be more appropriate

Reply:
make sense, will update